CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.h

```diff
@@ -34,8 +33,9 @@ class CaloTPGTranscoderULUT : public CaloTPGTranscoder {
   virtual double hcaletValue(const int& ieta, const int& iphi, const int& compressedValue) const;
   virtual double hcaletValue(const HcalTrigTowerDetId& hid, const HcalTriggerPrimitiveSample& hc) const;
   virtual bool HTvalid(const int ieta, const int iphi) const;
-  virtual std::vector<unsigned char> getCompressionLUT(HcalTrigTowerDetId id) const;
+  virtual std::vector<unsigned int> getCompressionLUT(HcalTrigTowerDetId id) const;
```

Comment:
thanks @fwyzard. Indeed the correct signature for a getter is to return a `const` reference:
````
virtual std::vector<unsigned int> const & getCompressionLUT(HcalTrigTowerDetId id) const
````